### DISCUSSION TITLE
Uno HotDesign limitations?

### DISCUSSION THREAD

**Discussion by DevTKSS** [unoplatform/studio#24] [Q&A] [answered]
- Like, do we have to stay with mvux or is that no problem when not using mvux like in guide?

**Comment by agneszitte** [comment, marked as answer]
Hi 👋 @DevTKSS, thank you for your question! 

You don’t need to stick with MVUX to use Hot Design. 
The MVVM pattern is also fully supported, as mentioned in our current documentation: https://aka.platform.uno/hot-design

![image](https://github.com/user-attachments/assets/56f2f869-372d-45fc-9d87-1db94b018da6)

Additionally, you can even start a new project without selecting any architectural pattern, such as MVVM or MVUX, and still take advantage of Hot Design, as it is also supported.

![image](https://github.com/user-attachments/assets/47a7b224-9fd6-41bd-9b97-d08831d302d2)

Please feel free to reach out with any additional questions on this topic. 

Thank you again for your feedback and for trying out Hot Design ❤️ !

**Reply by DevTKSS** [reply]
ah okay, when we might would want to add a mvvm getting started with HotDesign Guide, would that be wanted or not? Because it might be a bit hard for starters and I really like the guide you done for the mvux with the gif that are showing the way how.

**Reply by agneszitte** [reply]
@DevTKSS Thank you so much, your feedback is taken into account and changes will be published with an upcoming release to clarify the support outside of MVUX and to add additional details for MVVM as well.
